services/QuillLMS/client/app/bundles/Staff/components/evidence/activityStats/activityStats.tsx

```diff
@@ -114,8 +139,46 @@ const ActivityStats: React.FC<RouteComponentProps<ActivityRouteProps>> = ({ hist
     const percentageOptimalFirstAttempt = _.round(num_first_attempt_optimal / session_count * 100, 2)
     const percentageNotOptimalFirstAttempt = _.round(num_first_attempt_not_optimal / session_count * 100, 2)
 
+    let optimalAccuracyDisplay = '-';
+    let suboptimalAccuracyDisplay = '-';
+    let optimalResponsesCount = 0;
+    let suboptimalResponsesCount = 0;
+
+    const promptData = activityData?.activity?.prompts?.find((p: any) => p.text === display_name);
+    const conjunction = promptData?.conjunction;
+
+    if (conjunction && ruleFeedbackHistoriesMap && ruleFeedbackHistoriesMap[conjunction]) {
+      const conjunctionData = ruleFeedbackHistoriesMap[conjunction];
+
+      if (conjunctionData?.ruleFeedbackHistories) {
+        const genAIRules = Object.values(conjunctionData.ruleFeedbackHistories).filter(
+          (rule: any) => rule.api_name === 'genAI'
```

Comment:
We have `'genAI'` as the constant `GEN_AI` in the `constants/evidence` file, so I would import this from there.